spec/builtin_contracts_spec.rb

```diff
@@ -418,4 +418,41 @@ def something(hash)
       end
     end
   end
+
+  describe "StrictHash:" do
+    context "when given an exact correct input" do
+      it "does not raise an error" do
+        expect { @o.strict_person(:name => "calvin", :age => 10) }
```

Comment:
These lines are causing a syntax error in CI with Ruby 1.8.7: 
https://travis-ci.org/egonSchiele/contracts.ruby/jobs/123853792

You could do this as a block instead.